.env.example

```
@@ -10,114 +10,39 @@
+# match. Secrets are dev placeholders — do not use them in production.
 
 # ===========================================================================
 # Compose interpolation (host port mapping + postgres container init)
 # ===========================================================================
 
-# Postgres: initialises the db container AND injected into server + migrate.
+# Postgres: initialises the db container AND the migrate service's connection.
+# If you change DB_USER/DB_NAME here, also update db.postgres.user/name in
+# deployments/compose/config.yaml — that file can't read these vars.
 DB_USER=postgres
 DB_PASSWORD=postgres
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_PASSWORD key should go before the DB_USER key

(UnorderedKey)